conbench/tests/entities/test_history.py

```diff
@@ -386,3 +386,19 @@ def test_detect_shifts_with_trimmed_estimators():
     assert not np.any(result_df.is_step[:50])
     assert result_df.is_step[50]
     assert not np.any(result_df.is_step[51:])
+
+
+def test_set_z_scores_one_rep():
```

Comment:
Validated that this test would have failed before because the `z_score` was None. Now it passes.